Libraries/Microsoft.Teams.Api/Clients/UserTokenClient.cs

```diff
@@ -39,7 +38,7 @@ public UserTokenClient(IHttpClientFactory factory, CancellationToken cancellatio
     {
         var query = QueryString.Serialize(request);
         var req = HttpRequest.Post($"https://token.botframework.com/api/usertoken/GetAadTokens?{query}", body: request);
-        var res = await _http.SendAsync<IDictionary<string, TokenResponse>>(req, _cancellationToken);
+        var res = await _http.SendAsync<IDictionary<string, Token.Response>>(req, _cancellationToken);
         return (IDictionary<string, Token.Response>)res.Body;
```

Comment:
do we still need this cast?

Reply:
no we dont, wanted to confirm the change first.